### ISSUE TITLE
Error in generated output

### ISSUE THREAD

**Issue by mohammad-rahchamani** [MarcoEidinger/SwiftPlantUML#65]
I tried to generate the diagrams for a large project. I couldn't see the result in my browser.
So, I tried saving the console report in a file and then running `plantuml` to get the diagram.
But not it gives me this error:
`Error line 9428 in file: diagram.plant`
and here is that line:
`Handler<Void, [SelectedTenant]> <|-- FetchTenantHandler : inherits`
Is there anything wrong with this line?

**Comment by MarcoEidinger** [comment]
> Is there anything wrong with this line?

Yes, that is incorrect PlantUML syntax. What's the corresponding Swift code? I mean what's the Swift definition of `Handler`?

Generics are supported by SwiftPlantUML but my implementation might have a bug. The following simplified example

```swift
class Stack<Element> {}

class SpecificStack: Stack<Int> {}
```

will be generated correctly into PlantUML

```
@startuml
' STYLE START
hide empty members
skinparam shadowing false
' STYLE END


class "Stack" as Stack<Element> << (C, DarkSeaGreen) >> {  
}
class "SpecificStack" as SpecificStack<Int> << (C, DarkSeaGreen) >> {  
}
Stack <|-- SpecificStack : inherits

@enduml
```

and can then be visualized

![](https://www.planttext.com/api/plantuml/png/XO-n2i9044Jx_Oh1HWNp0tA4Y0OHnCAaiLoJrHp9dU7sH4JzTyL2jB8PXc7Um6GQESIpwsY8ejojyvVEjYKrjXQ8w-CLJjnUWfAsrlSSs44RhayNwuyuSAVoGVFDWeYgZbKnA29NxG2iU3kJT-B4nnJ6O3IVOC6XBOINGSIFaQQu0VJuqBrKzc2hhvNlnAp-BxsBCFSa-MKnXVMD11kLA1DVlzu_0G00)

**Comment by MarcoEidinger** [comment]
@mohammad-rahchamani can you tell the corresponding Swift code, i.e the Swift definition of `Handler`?

**Comment by mohammad-rahchamani** [comment]
@MarcoEidinger Thanks for the explanation. 
Here is the `Handler` class definition:

```

open class Handler<T, S> {
    
    public init() { }
    
    open func handle(input: T, completion: @escaping (Result<S, Error>) -> Void) {
        fatalError("this method should be implemented in child class.")
    }
    
}
```

**Comment by MarcoEidinger** [comment]
@mohammad-rahchamani thanks, I am able to reproduce the issue with the following and will look into the root cause in the next couple of days

```
open class Handler<T, S> {
    public init() { }
    open func handle(input: T, completion: @escaping (Result<S, Error>) -> Void) {
        fatalError("this method should be implemented in child class.")
    }
}

class FetchTenantHandler: Handler<Void, [SelectedTenant]> {}

struct SelectedTenant {}
```

In the meanwhile you can fix the diagram manually by adjusting the PlantUML script from

```
class "Handler" as Handler<T\nS> << (C, DarkSeaGreen) >> { 
  +init()
  +handle(input:completion:) 
}
class "FetchTenantHandler" as FetchTenantHandler << (C, DarkSeaGreen) >> {  
}
Handler<Void, [SelectedTenant]> <|-- FetchTenantHandler : inherits
```

to

```
class "Handler" as Handler<T\nS> << (C, DarkSeaGreen) >> { 
  +init()
  +handle(input:completion:) 
}
class "FetchTenantHandler" as FetchTenantHandler<Void, [SelectedTenant]> << (C, DarkSeaGreen) >> {  
}
Handler <|-- FetchTenantHandler : inherits
```

which will result in

![](https://www.planttext.com/api/plantuml/png/VP11IyH038Nl-HMFlTZYxXyeGrbnLps81ri4KG-X4-smqtIPoU91_UzsTHL1yH8UUNdvGfRPEDbkY7I4fhszN4_rvBebofkWMquy8FVin-UWJtZacELxS7sr8g8kSiwONR3wA6a6pZXerzvhKyCv5ATph3Xj6k7p9A8bwXel8E0uQB2YtAl-8rLCr9rLtJXierWOjIf1Rr-KCx6kRqLPxIVmTzlTZC7FSTT8bCx4VveF_rspnnpISA-BnHzRKI5eBobO9bgA-kblxm00)

**Comment by mohammad-rahchamani** [comment]
That sounds wonderful. 
I'll try the workaround.
Thank you.

**Comment by MarcoEidinger** [comment]
@mohammad-rahchamani you can also configure to ignore generics so that such information will not be shown (showGenerics:false). Then diagram generation should work without problems. See README and configuration section for details

**Comment by MarcoEidinger** [comment]
Found the root cause and I probably can release 0.7.4 with respective bug fix in the next 1-2 days

**Comment by MarcoEidinger** [comment]
@mohammad-rahchamani fix is available in 0.7.4. Homebrew update might still be [pending](https://github.com/Homebrew/homebrew-core/pull/123565) if you check
